src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -649,8 +649,13 @@ await AddDependencyFlowEventsAsync(
             MergePolicyCheckResult.PendingPolicies,
             pr.Url);
 
+        // When changing `from` and `to` for depenency updates in a PR's description, we want `from` to keep
+        // pointing to the original version from the target branch.
```

Comment:
Oh the comment is here, but maybe it would be better with the method

Reply:
Moved it to the method